OneCommunityGlobal/HighestGoodNetworkApp
# Peterson - Fix the issue causing the user profile page to display a white screen upon  access

# Description
This pull request has been opened to fix the bug that causes the screen to appear white and multiple errors to appear in the browser console.

## Related PRS (if any):
none
…

## Main changes explained:
The user profile  component has been modified to fix the bug.
…

## How to test:
check into current branch
do npm install and ... to run this PR locally
Log in with any account that hasn't saved the start date:
![photo 2](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/73541319/acbcfee7-89a1-48c3-a21e-c8efeae35f1d)

go to view profile
The user profile page should render all user information, and the errors shown in the image below shouldn't be displayed in the browser console:
![photo](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/73541319/c9fb8b97-cad3-469a-9874-436e7bcf9447)

## Screenshots or videos of changes:
Before my fix:
![a](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/73541319/9cb78e6d-6c83-42b5-ae55-cac5a06ece52)

After my fix: 
![b](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/73541319/5298dad2-c6a1-4f13-83f7-f6855b78e76d)

## Note:
none
